plugins/rtmp-services/data/schema/service-schema-v5.json

```diff
@@ -198,6 +198,14 @@
                         "$ref": "#/definitions/saneUrl",
                         "description": "Link that provides additional info about the service, presented in the UI as a button next to the services dropdown."
                     },
+                    "ertmp_configuration_url": {
+                        "$ref": "#/definitions/saneUrl",
+                        "description": "Accessed for multitrack video auto configuration"
+                    },
+                    "ertmp_configuration_name": {
```

Comment:
Ideally, any user-facing text should be supported by our translation system. As this cannot be, this should not be configurable text, but rather a generic platform/protocol-agnostic field title in our translation file.

Reply:
Putting this text in our translation system would require jumping through extra hoops, or us having to merge and maintain strings in our codebase. I'm not sure how much extra work that would entail, but I do not consider this feedback a blocker at this time.